### PR TITLE
OTT-105: VCR - Video Event Trackers

### PR DESCRIPTION
**Pull request by ShriprasadM**

### CODE REVIEW

File: adapters/bidder.go [hunk 1]
```diff
@@ -118,10 +118,11 @@ type ResponseData struct {
 
 // RequestData packages together the fields needed to make an http.Request.
 type RequestData struct {
-	Method  string
-	Uri     string
-	Body    []byte
-	Headers http.Header
+	ImpIndex int
```
Comment (pm-viral-vala) [inline adapters/bidder.go:121]: Please remove ImpIndex its VAST Bidder changes

File: adapters/tagbidder/bidder_config.go.bak [hunk 2]
```diff
@@ -0,0 +1,70 @@
+package tagbidder
```
Comment (pm-viral-vala) [inline adapters/tagbidder/bidder_config.go.bak:1]: Please remove adapters/tagbidder folder its VAST Bidder changes

File: router/router_test.go [hunk 3]
```diff
@@ -54,7 +54,7 @@ func TestNewJsonDirectoryServer(t *testing.T) {
 	}
 
 	for _, adapterFile := range adapterFiles {
-		if adapterFile.IsDir() && adapterFile.Name() != "adapterstest" {
+		if adapterFile.IsDir() && adapterFile.Name() != "adapterstest" && adapterFile.Name() != "tagbidder" {
```
Comment (pm-viral-vala) [inline router/router_test.go:57]: Remove this is VAST Bidder Changes

File: static/tagbidder-params/spotx.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (pm-viral-vala) [inline static/tagbidder-params/spotx.json:1]: Remove static/tagbidder-params folder

**Review by pm-viral-vala — CHANGES_REQUESTED** [review]

File: adapters/bidder.go (hunk 1 above)
Comment (ShriprasadM) [reply adapters/bidder.go]: Done

File: router/router_test.go (hunk 3 above)
Comment (ShriprasadM) [reply router/router_test.go]: Done

File: adapters/tagbidder/bidder_config.go.bak (hunk 2 above)
Comment (ShriprasadM) [reply adapters/tagbidder/bidder_config.go.bak]: done

File: static/tagbidder-params/spotx.json (hunk 4 above)
Comment (ShriprasadM) [reply static/tagbidder-params/spotx.json]: Done

**Review by pm-viral-vala — APPROVED** [review]
LGTM

**Comment by ShriprasadM** [comment]
Checks are failing because of existing prebid-server testcases. e.g. syncer.go , adaptors/bidder.go

**Pull request merged**